english/exercise/exercise.md

```diff
@@ -0,0 +1 @@
+If you [eat too much mashmallows](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md), you will gain weight. Therefore, be sure to go for a run and do some exercise, lose some calories and then [eat](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md) again.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.